tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -1240,4 +1245,200 @@ public function checkUserInformation(array $expectedValue, array  $actualValue):
 			}
 		}
 	}
+
+	/**
+	 * @When user :user tries to get information of user :ofUser using Graph API
+	 *
+	 * @param string $user
+	 * @param string $ofUser
```

Comment:
We have some already implemented steps, using variables like `byUser` for an acting user and `user` for a target user. It will be better if we use consistent variable names.